auction_keeper/main.py

```diff
@@ -284,8 +285,9 @@ def startup(self):
             logging.info("Keeper will perform the following operation(s) in parallel:")
             [logging.info(line) for line in notice_string]
 
-            logging.info("*** When Keeper is dealing/bidding, the initial evaluation of auctions will likely take > 45 minutes without setting a lower boundary via '--min-auction' ***")
-            logging.info("*** When Keeper is kicking, the recurring query of Vaults will likely take > 30 minutes each loop without using VulcanizeDB via `--vulcanize-endpoint` ***")
+            if self.flipper and self.ilk and self.ilk.name == "ETH-A":
+                logging.info("*** When Keeper is dealing/bidding, the initial evaluation of auctions will likely take > 45 minutes without setting a lower boundary via '--min-auction' ***")
+                logging.info("*** When Keeper is kicking, the recurring query of Vaults will likely take > 30 minutes each loop without using VulcanizeDB via `--vulcanize-endpoint` ***")
```

Comment:
@KentonPrescott  These warnings only apply to ETH, and that's on a slow node.  BAT kicking will take similar length, but bidding will be much quicker because there have been less auctions.  WBTC should be very quick so long as the starting block is set to sometime around this comment.

Reply:
I admit the implementation of the startup warning logic is suboptimal and wonder if there's a way to estimate the wait periods based on a number of factors (such as auction number, or collateral-addition block, etc), or source a community hosted vulcanizeDB endpoint. 

There isn't a solution that shines above all and has a clear path forward. I'll pick this back up if any feedback, relating to performance issues (as a result of misunderstood configs), comes up.